plugins/linux-v4l2/v4l2-output.c

```diff
@@ -77,8 +87,11 @@ static bool loopback_module_loaded()
 		}
 	}
 
-	fclose(fp);
-
+	if (!fclose(fp)) {
+		blog(LOG_WARNING,
```

Comment:
See above, the level should depend on whether it it possible for `fclose` to fail under normal circumstances.

E.g. if the OS itself can close the file descriptor at any point and there is no way for us to notice and this is a "normal" occurrence on Linux.